test/tests/api/v2_0/nodes_tests.py

```diff
@@ -80,24 +80,22 @@ def __init__(self):
     def __get_data(self):
         return loads(self.__client.last_response.data)
 
-    def __get_workflow_status(self, id):
-        Api().nodes_get_active_workflow_by_id(identifier=id)
-        status = self.__client.last_response.status
-        if status == 200:
-            data = self.__get_data()
-            if data:
-                status = data.get('_status')
-                assert_is_not_none(status)
-        return status
+    def __get_workflow_status(self, id, query ):
+	Api().nodes_get_workflow_by_id(identifier=id, active=query )
+	data = self.__get_data()
+	if len(data) > 0:
+	    status = data[0].get('_status')
+	    return status
+	return 'not running'
         
     def __post_workflow(self, id, graph_name):
-        status = self.__get_workflow_status(id)
+        status = self.__get_workflow_status(id, 'true')
```

Comment:
@yyscamper when I tried passing it in as a Boolean it threw errors.  I believe the Boolean is lowercase in python and capitalized in javascript which was messing things up